materials/sql_joins.md

```diff
@@ -7,44 +7,65 @@ language: SQL
 
 ### Basic join
 
+* [`JOIN`](http://www.w3schools.com/sql/sql_join.asp) 
+    * combine rows from multiple tables
+    * based on condition
+* This query selects `species` records in the `PortalMammals_main` table and 
+  `scientificname` found in the `PortalMammals_species` table.
+    * The query links the `species` with `new_code` from `PortalMammals_species`.
+  
 ```
-SELECT yr, mo, dy, scientificname
-FROM Main
-JOIN Species ON species = new_code;
+SELECT species, scientificname
+FROM PortalMammals_main
+JOIN PortalMammsls_species WHERE species = new_code;
```

Comment:
I recall reading somewhere not to show the wrong way or at least not to show it first, but I don't remember where so I'm not sure if my brain is just inventing that recollection.